### PR TITLE
chore: knobs added to save and share bar

### PR DESCRIPTION
**Pull request by infreezer**
image

### CODE REVIEW

**Comment by aronshamash** [comment]
@infreezer do we need to add any more snapshots for these scenarios too ?

**Comment by infreezer** [comment]
> @infreezer do we need to add any more snapshots for these scenarios too ?

Sure. We can do it in another PR and merge this one.

File: packages/save-and-share-bar/package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "@times-components/tracking": "2.5.2",
     "@times-components/user-state": "0.1.5",
     "@times-components/utils": "5.1.9",
+    "@storybook/addon-knobs": "4.1.18",
```
Comment (mikefieldmay) [inline packages/save-and-share-bar/package.json]: Just to double check, this shouldn't have caused any changes to the yarn.lock should it?

Comment (infreezer) [reply packages/save-and-share-bar/package.json]: No changes in yarn.lock

**Pull request merged**